xdsl/rewriter.py

```diff
@@ -255,22 +257,24 @@ def insert_block_before(block: Block | list[Block], target: Block):
         region.insert_block(block_list, pos)
 
     @staticmethod
-    def insert_ops_at_location(ops: Sequence[Operation], insertion_point: InsertPoint):
+    def insert_ops(ops: Sequence[Operation], insertion_point: InsertPoint):
         """Insert operations at a certain location in a block."""
         if insertion_point.insert_before is not None:
             insertion_point.block.insert_ops_before(ops, insertion_point.insert_before)
         else:
             insertion_point.block.add_ops(ops)
```

Comment:
Nevermind, I will comment on the naming. Can we have this be `insert_op(op: Operation | Sequence[Operation], ...) to mirror the other API?

Reply:
Messaged in Zulip to ask whether to do this before or after this one is in, or as part of this PR